introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification_Benchmarking.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>We've had a request from ADC to not do this as it can break their environments when you get an upgrade (particularly upsets numpy). Can you remove this and still pass CI?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Can you be more specific? I'm not sure which cell this comment is referring to.